### DISCUSSION TITLE
when i exec fyne package -os js ,i reminds me :unsupported target operating system "js",why is that.thanks ![image](https://github.com/user-attachments/assets/682a7659-37c1-41ef-b058-6ccb440a6ad7)

### DISCUSSION THREAD

**Discussion by GodGavin** [fyne-io/fyne#5049] [Q&A]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same problem, before opening a new one.
- [X] This issue only relates to a single bug. I will open new issues for any other problems.

### Describe the bug

when i exec fyne package -os js ,i reminds me :unsupported target operating system "js",why is that.thanks
![image](https://github.com/user-attachments/assets/682a7659-37c1-41ef-b058-6ccb440a6ad7)

### How to reproduce

when i exec fyne package -os js ,i reminds me :unsupported target operating system "js",why is that.thanks
![image](https://github.com/user-attachments/assets/682a7659-37c1-41ef-b058-6ccb440a6ad7)

### Example code

when i exec fyne package -os js ,i reminds me :unsupported target operating system "js",why is that.thanks
![image](https://github.com/user-attachments/assets/682a7659-37c1-41ef-b058-6ccb440a6ad7)

### Fyne version

latest

### Go compiler version

1.22.2

### Operating system and version

win

### Additional Information

Win10

**Comment by andydotxyz** [comment]
This seems like a question not a bug report, so moving to discussions.

**Comment by andydotxyz** [comment]
There is no "js" target system - we use wasm / web since v2.5.0.

What are you trying to achieve the is looking for a JS specific runtime?

**Reply by GodGavin** [reply]
The doc said it can, i says u can specifiy js or wasm
And i think the wasm file is too large,it should be better have a js

**Reply by GodGavin** [reply]
> There is no "js" target system - we use wasm / web since v2.5.0.
> 
> What are you trying to achieve the is looking for a JS specific runtime?

And u said The WASM package that runs a web driver does not currently have access to scan the user's font directories for platform fonts
 Js can solve that?

**Reply by andydotxyz** [reply]
> The doc said it can, i says u can specifiy js or wasm And i think the wasm file is too large,it should be better have a js

I think the doc may have been out of date - this was removed a few days ago, the "js" runtime was a temporary thing whilst WASM support in some browsers was not good enough. It has been removed.

> And u said The WASM package that runs a web driver does not currently have access to scan the user's font directories for platform fonts Js can solve that?

They both have the same security access, and the WASM package can call out to JavaScript to access any APIs that exist, so it won't make any difference.
